addons/binding/org.openhab.binding.blebox/ESH-INF/thing/dimmerBox.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="blebox"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+    xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+    <thing-type id="dimmerBox">
+
+        
+        <label>dimmerBox</label>
+        <description>Blebox dimmerBox</description>
+
+        <channels>
+            <channel id="brightness" typeId="brightness" />
+        </channels>
+
+        <config-description>
```

Comment:
If you have a lot of the same configs you can use `<config-description-ref =...`  and point to a single config for all things.